qiskit_machine_learning/utils/loss_functions/loss_functions.py

```diff
@@ -13,46 +13,95 @@
 """ Loss utilities """
 
 from abc import ABC, abstractmethod
+from typing import Union, Tuple
 import numpy as np
 
+
 from ...exceptions import QiskitMachineLearningError
 
 
 class Loss(ABC):
     """
-    Abstract base class for Loss.
+    Abstract base class for computing Loss.
+
     """
 
-    def __call__(self, predict, target):
+    def __call__(self, predict: Union[int, np.ndarray], target: Union[int, np.ndarray]) -> float:
```

Comment:
So this is now either an integer or an numpy array.  The args below mention just the numpy array not a single value. I see the single value is reformed to a numpy array. Does that mean the numpy arrays are arrays of ints too? I.e the predicted and target values are all ints. Maybe the Args docstring can be improved.

Reply:
> 
> 
> So this is now either an integer or an numpy array. The args below mention just the numpy array not a single value. I see the single value is reformed to a numpy array. Does that mean the numpy arrays are arrays of ints too? I.e the predicted and target values are all ints. Maybe the Args docstring can be improved.

No more ints, just numpy arrays.